docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md

```diff
@@ -381,18 +382,31 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
             Both gateway-url and service-url parameters specify how the API service endpoints are mapped to the API
             gateway endpoints. The service-url parameter points to the target endpoint on the gateway.
     
-    * **apiInfo.title**
-    
-        Specifies the title of your service API.
-    
-    * **apiInfo.description**
-    
-        Specifies the high-level function description of your service API.
-    
-    * **apiInfo.version**
-    
-        Specifies the actual version of the API in semantic format.
-    
+    * **apiInfo.apiId**
+
+        Specifies the API identifier that is registered in the API Mediation Layer installation.
+        The API ID uniquely identifies the API in the API Mediation Layer.
+        The same API can be provided by multiple services. The API ID can be used
+        to locate the same APIs that are provided by different services.
+        The creator of the API defines this ID.
+        The API ID needs to be a string of up to 64 characters
+        that uses lowercase alphanumeric characters and a dot: `.`.
+        We recommended you use your organization as the prefix.
+
+    * **apiInfo.gatewayUrl**
+
+        The base path at the API Gateway where the API is available. It should be
+        the same as a _gatewayUrl_ value in the _routes_ sections.
+
+    * **apiInfo.swaggerUrl**
+
+        (Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document
+        that provides the API documentation for this API is available.
+        
+    * **apiInfo.documentationUrl**
+
+        (Optional) Link to external documentation, if needed. Can coexist with Swagger documentation.
```

Comment:
Explain 'coexist'. Can this be re-written? 
Example:
"the link to the external documentation can be included along with the Swagger documentation"

Reply:
I replaced it